### PR TITLE
Swallow error during test

### PR DESCRIPTION
**Pull request by nuernbergerA**
When faking `tmp-for-tests` a error is thrown that is handled but also bubbles up and leads to "ugly" output

```bash
[2024-07-17 13:08:13] testing.ERROR: Disk [tmp-for-tests] does not have a configured driver. {"exception":"[object] (InvalidArgumentException(code: 0): Disk [tmp-for-tests] does not have a configured driver. at /home/adrian/code/bauservices/vendor/laravel/framework/src/Illuminate/Filesystem/FilesystemManager.php:137)
[stacktrace]
...
```

This happens in a edgy-edge case when you have to call this in you test
```php
FileUploadConfiguration::storage();
TemporaryUploadedFile::createFromLivewire(...)
```

i've separated all refactoring steps into atomic commits:
* actual fix
* fixed a typo
* used arrow functions (available sind PHP 7.4)
* extracted disk to a variable (was called 3 times)

### CODE REVIEW

File: src/Features/SupportFileUploads/FileUploadConfiguration.php [hunk 1]
```diff
@@ -9,19 +9,22 @@ class FileUploadConfiguration
...
+            rescue(
                 // If the storage disk is not found (meaning it's the first time),
                 // this will throw an error and trip the second callback.
-                return Storage::disk(static::disk());
-            }, function () {
-                return Storage::fake(static::disk());
-            });
+                fn() => Storage::disk($disk),
+                fn() => Storage::fake($disk),
+                // swallows the error that is thrown on the frist try
```
Comment (edalzell) [inline src/Features/SupportFileUploads/FileUploadConfiguration.php:22]: 'first'

**Comment by edalzell** [comment]
Love this one Adrian, thanks

File: src/Features/SupportFileUploads/FileUploadConfiguration.php (hunk 1 above)
Comment (nuernbergerA) [reply src/Features/SupportFileUploads/FileUploadConfiguration.php:22]:
```suggestion
                // swallows the error that is thrown on the first try
```

**Comment by nuernbergerA** [comment]
@calebporzio pls merge
